packages/neuron-wallet/tests/controllers/sudt.test.ts

```diff
@@ -63,8 +63,8 @@ describe('SUDTController', () => {
       'txHash',
       '0',
       '10000',
-      Script.fromObject({ codeHash: '', args: '', hashType: ScriptHashType.Type }),
-      Script.fromObject({ codeHash: '', args: '', hashType: ScriptHashType.Type }),
+      Script.fromObject({ codeHash: `0x${'00'.repeat(32)}`, args: '0x', hashType: ScriptHashType.Type }),
```

Comment:
it is better to add some comment to describe why the empty `codeHash` is required here

Reply:
I have described the reason in the PR comment, please refer to:https://github.com/nervosnetwork/neuron/pull/2734#issue-1765073549